src/app/components/messages/messages.component.html

```diff
@@ -16,7 +16,7 @@
         <mat-icon inline [matTooltip]="tooltip(item?.status)" [fontIcon]="icon(item?.status)" class="icon"></mat-icon>
       </div>
     </div>
-    @if ($index !== filteredMessages.length - 1) {
+    @if ($index !== filteredMessages().length - 1) {
```

Comment:
Improves readability.

```suggestion
    @if ($index < filteredMessages().length - 1) {
```